installers/ansible/inventory

```diff
@@ -26,6 +30,14 @@ crunchy_debug='false'
 # ==================
 # kubernetes_context=''
 
+# Deploy with PGO-Installer
```

Comment:
` # Deploy with PostgreSQL Operator Installer (pgo-installer)`

Let's also put this above the other two methods, try to encourage people to use this method